src/Core/Checkout/Document/Renderer/InvoiceRenderer.php

```diff
@@ -140,17 +147,40 @@ public function render(array $operations, Context $context, DocumentRendererConf
                         $context,
                         $order->getSalesChannelId(),
                         $order->getLanguageId(),
-                        $locale->getCode()
+                        $locale->getCode(),
                     );
 
                     $doc = new RenderedDocument(
-                        $html,
+                        $html, // @deprecated tag:v6.7.0 - html argument will be removed
                         $number,
                         $config->buildName(),
                         $operation->getFileType(),
                         $config->jsonSerialize(),
                     );
 
+                    // set the template renderer to be able to render the a11y version
+                    $doc->setTemplateOptions([
+                        $template,
+                        [
+                            'order' => $order,
+                            'config' => $config,
+                            'rootDir' => $this->rootDir,
+                            'context' => $context,
+                        ],
+                        $context,
+                        $order->getSalesChannelId(),
+                        $order->getLanguageId(),
+                        $locale->getCode(),
+                    ]);
+
+                    if ($operation->getFileType() === HtmlRenderer::FILE_EXTENSION) {
```

Comment:
seeing the duplication in all those renderers I question even more if this is actually the right place to actually render the files 🤔 see: https://github.com/shopware/shopware/pull/5911/files#r1905438687